ffi/gn/icu4x/BUILD.gn

```diff
@@ -254,9 +235,6 @@ rust_library("icu_capi-v1_2_2") {
   deps += [ ":tinystr-v0_7_1" ]
   deps += [ ":unicode-bidi-v0_3_13" ]
   deps += [ ":writeable-v0_5_2" ]
-  if (current_cpu == "wasm32") {
-    deps += [ ":log-v0_4_17" ]
-  }
```

Comment:
Observation: Was logging ever enabled by default in wasm? The `log` crate should be a dependency of `icu_provider`, not `diplomat`

Reply:
And `icu_capi` itself needs the `log` crate to register the wasm console logger. I'd like to move that into `diplomat_runtime` at some point, but for now it's here.

Reply:
And `icu_capi` itself needs the `log` crate to register the wasm console logger. I'd like to move that into `diplomat_runtime` at some point, but for now it's here.